### ISSUE TITLE
Dude, how did you manage to write these materials?

### ISSUE THREAD

**Issue by beef9999** [shichao-an/notes#2]
Are they collected from the Internet or created by yourself originally?

**Comment by shichao-an** [comment]
Hi, I collected or summarized them from books and online materials provided by others. I didn't create them originally.

**Comment by ajarx** [comment]
Hi, I read the UNP pages about Unix network programming, is it a existing book or your summarize from a book?
Anyway, I like it, it helps a lot. Thanks!

**Comment by shichao-an** [comment]
@tree1224 Yeah, I summarized most content from the book. Glad to hear that it helps!

**Comment by devt3000** [comment]
Thanks 😎👌! Are you on Twitter?
